terraform/modules/consumer/lambdas/get_event.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import os
+from datetime import datetime
+from http.client import HTTPResponse
+from urllib import request
+
+from common import get_auth_token
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+gdx_url = os.environ["gdx_url"]
+events_url = gdx_url + "/events"
+
+auth_url = os.environ["auth_url"]
+
+client_id = os.environ["client_id"]
+client_secret = os.environ["client_secret"]
+
+lev_api_url = os.environ["lev_api_url"]
```

Comment:
I find this surprising, I'd have assumed that our event consumption is ignorant of anything but GDX, are we doing some validation of things against LEV?

Reply:
I scrolled a bit further on and found the answer. I think this is worth an explanatory note though, in that normal consumers would not need to do this.